.ci/run-tests

```diff
@@ -10,4 +10,5 @@
 #   - $NODE_JS_VERSION
 #
 
-ELASTICSEARCH_VERSION=${ELASTICSEARCH_VERSION} NODE_JS_VERSION=${NODE_JS_VERSION} docker-compose -f .ci/docker-compose.yml run client
+ELASTICSEARCH_VERSION=${ELASTICSEARCH_VERSION} NODE_JS_VERSION=${NODE_JS_VERSION} docker-compose -f .ci/docker-compose.yml run client-oss
+ELASTICSEARCH_VERSION=${ELASTICSEARCH_VERSION} NODE_JS_VERSION=${NODE_JS_VERSION} docker-compose -f .ci/docker-compose.yml run client-commercial
```

Comment:
I'm not sure how I feel about this. 
one one hand it works... 
one another the jenkins worker isn't really isolating the test. 

Maybe we can do some work with the test matrix? To add a another "element" to the matrix?

My point being if a test fails in the commertial api but dos not fail in the oss lib then we'll get a failure in jenkins and it won't be obviously clear which one failed...

Reply:
This morning @karmi made the same point and I agree completely.
How can we achieve that?